### PR TITLE
geolocation-API: Feature-Policy is now Permissions-Policy

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

File: geolocation-API/disabled-by-permissions-policy.https.sub.html [hunk 1]
```diff
@@ -10,9 +10,9 @@
     "use strict";
 
     const same_origin_src =
-      "/feature-policy/resources/feature-policy-geolocation.html";
+      "/feature-policy/resources/permissions-policy-geolocation.html";
     const cross_origin_src =
-      "https://{{domains[www]}}:{{ports[https][0]}}" + same_origin_src;
```
Comment (reillyeon) [inline geolocation-API/disabled-by-permissions-policy.https.sub.html:15]: At least on the version of wptserve Chromium is using this causes the server to throw an exception. Are you sure this is the correct syntax? What was wrong with the existing code?

File: geolocation-API/disabled-by-permissions-policy.https.sub.html [hunk 2]
```diff
@@ -10,9 +10,9 @@
...
+      "https://{{hosts[]}}:{{ports[https][0]}}" + same_origin_src;
```
Comment (marcoscaceres) [inline geolocation-API/disabled-by-permissions-policy.https.sub.html:15]:
```suggestion
      "https://{{hosts[][]}}:{{ports[https][0]}}" + same_origin_src;
```

File: geolocation-API/disabled-by-permissions-policy.https.sub.html (hunk 1 above)
Comment (marcoscaceres) [reply geolocation-API/disabled-by-permissions-policy.https.sub.html:15]: oops, indeed... should have been https://{{hosts[][]}}

File: geolocation-API/enabled-by-permissions-policy.https.sub.html [hunk 3]
```diff
@@ -8,9 +8,9 @@
   <script src="/feature-policy/resources/featurepolicy.js"></script>
   <script>
     const same_origin_src =
-      "/feature-policy/resources/feature-policy-geolocation.html";
+      "/feature-policy/resources/permissions-policy-geolocation.html";
     const cross_origin_src =
-      "https://{{domains[www]}}:{{ports[https][0]}}" + same_origin_src;
+      "https://{{hosts[]}}:{{ports[https][0]}}" + same_origin_src;
```
Comment (marcoscaceres) [inline geolocation-API/enabled-by-permissions-policy.https.sub.html:13]:
```suggestion
      "https://{{hosts[][]}}:{{ports[https][0]}}" + same_origin_src;
```

File: geolocation-API/enabled-by-permissions-policy.https.sub.html.headers [hunk 4]
```diff
@@ -0,0 +1 @@
+Permissions-Policy: geolocation *
```
Comment (marcoscaceres) [inline geolocation-API/enabled-by-permissions-policy.https.sub.html.headers:1]:
```suggestion
Permissions-Policy: geolocation=*
```

File: geolocation-API/enabled-on-self-origin-by-permissions-policy.https.sub.html [hunk 5]
```diff
@@ -10,9 +10,9 @@
     "use strict";
 
     const same_origin_src =
-      "/feature-policy/resources/feature-policy-geolocation.html";
+      "/feature-policy/resources/permissions-policy-geolocation.html";
     const cross_origin_src =
-      "https://{{domains[www]}}:{{ports[https][0]}}" + same_origin_src;
+      "https://{{hosts[]}}:{{ports[https][0]}}" + same_origin_src;
```
Comment (marcoscaceres) [inline geolocation-API/enabled-on-self-origin-by-permissions-policy.https.sub.html:15]:
```suggestion
      "https://{{hosts[][]}}:{{ports[https][0]}}" + same_origin_src;
```

File: geolocation-API/disabled-by-permissions-policy.https.sub.html [hunk 6]
```diff
@@ -10,9 +10,9 @@
     "use strict";
 
     const same_origin_src =
-      "/feature-policy/resources/feature-policy-geolocation.html";
+      "/feature-policy/resources/permissions-policy-geolocation.html";
```
Comment (marcoscaceres) [inline geolocation-API/disabled-by-permissions-policy.https.sub.html:13]:
```suggestion
      "/permissions-policy/resources/permissions-policy-geolocation.html";
```

Comment (marcoscaceres) [inline geolocation-API/disabled-by-permissions-policy.https.sub.html:13]:
```suggestion
      "/permissions-policy/resources/permissions-policy-geolocation.html";
```

File: geolocation-API/disabled-by-permissions-policy.https.sub.html [hunk 7]
```diff
@@ -10,9 +10,9 @@
     "use strict";
 
     const same_origin_src =
-      "/feature-policy/resources/feature-policy-geolocation.html";
+      "/permissions-policy/resources/permissions-policy-geolocation.html";
     const cross_origin_src =
-      "https://{{domains[www]}}:{{ports[https][0]}}" + same_origin_src;
+      "https://{{hosts[]}}:{{ports[https][0]}}" + same_origin_src;
```
Comment (marcoscaceres) [inline geolocation-API/disabled-by-permissions-policy.https.sub.html:15]:
```suggestion
      "https://{{hosts[][]}}:{{ports[https][0]}}" + same_origin_src;
```

File: geolocation-API/enabled-by-permissions-policy.https.sub.html (hunk 3 above)
Comment (marcoscaceres) [inline geolocation-API/enabled-by-permissions-policy.https.sub.html:13]:
```suggestion
      "https://{{hosts[][]}}:{{ports[https][0]}}" + same_origin_src;
```

File: geolocation-API/enabled-on-self-origin-by-permissions-policy.https.sub.html (hunk 5 above)
Comment (marcoscaceres) [inline geolocation-API/enabled-on-self-origin-by-permissions-policy.https.sub.html:15]:
```suggestion
      "https://{{hosts[][]}}:{{ports[https][0]}}" + same_origin_src;
```

**Comment by marcoscaceres** [comment]
I didn't realize these tests were so busted. 

There is still a problem in Chrome... set_permission("geolocation") never resolves in some tests for some unknown reason.

File: geolocation-API/enabled-by-permission-policy-attribute-redirect-on-load.https.sub.html [hunk 8]
```diff
@@ -20,17 +20,17 @@
       expect_feature_available_default,
       "geolocation"
     );
-  }, 'Feature-Policy allow="geolocation" allows same-origin relocation');
+  }, 'Permissions-Policy allow="geolocation" allows same-origin geolocation');
```
Comment (marcoscaceres) [inline geolocation-API/enabled-by-permission-policy-attribute-redirect-on-load.https.sub.html:23]:
```suggestion
  }, 'Permissions-Policy allow="geolocation" allows same-origin relocation');
```

File: geolocation-API/enabled-by-permission-policy-attribute-redirect-on-load.https.sub.html [hunk 9]
```diff
@@ -20,17 +20,17 @@
...
     test_feature_availability(
       'geolocation',
       t,
       cross_origin_src,
-      expect_feature_unavailable_default,
+      expect_feature_available_default,
       "geolocation"
     );
-  }, 'Feature-Policy allow="geolocation" disallows cross-origin relocation');
+  }, 'Permissions-Policy allow="geolocation" allows cross-origin geolocation');
```
Comment (marcoscaceres) [inline geolocation-API/enabled-by-permission-policy-attribute-redirect-on-load.https.sub.html:33]:
```suggestion
  }, 'Permissions-Policy allow="geolocation" allows cross-origin relocation');
```

**Review by rakuco — APPROVED** [review]
lgtm

**Pull request merged**